src/app/services/client/client.service.ts

```diff
@@ -89,8 +89,8 @@ export interface AgencyStatus {
 @Injectable()
 export class ClientService {
 
-  private BASE: string = api.base;
-  private KEY: string = process.env.CODE_GOV_API_KEY || api.key;
+  private BASE: string = api.base || 'http://localhost:3000/';
```

Comment:
@froi , I know it's minor, but could you just specify http://localhost:300/ as the base in the [config file]( https://github.com/GSA/code-gov-web/blob/master/config/examples/code-dot-gov.json).  I would prefer to keep as much configuration as possible in one spot.  Everything else looks good.

Reply:
Sure. I do have a recommendation/comment about that. 

I think this needs to be more defensive. There is no validation or "good" feedback for a user that does not include a value in the config.json. The app also behaves very differently when you have some sort of default vs not having a value.

Eg.

Removing or leaving the `base` field blank does not throw any error during build and give a console error that does not really help the user figure out what's going on:

![no-default-base](https://user-images.githubusercontent.com/1918027/44594486-5225af80-a794-11e8-961d-72b567ef2fa3.png)

I think we should think a bit on how to make the dev experience "better" and maybe rethink some things about how we are managing our configs in all of our projects. I'll make an issue on code-gov or maybe a new issue on each project to discuss this.

Thanks for the review.